CP/Assignment6-Bruteforce/Chaitany/402B.cpp

```diff
@@ -0,0 +1,42 @@
+#include<bits/stdc++.h>
+using namespace std;
+int main()
+{
+
+    int n,k;
+    cin>>n>>k;
+    int arr[n];pair <int,int> p[1001]={make_pair(10000,1000)};
+    int time[1000];
+    for(int i=0;i<n;i++)
+    {
+        cin>>arr[i];
+    }
+    for(int j=1;j<=1000;j++)
+    {int ti=0;
+
+        for(int t=0;t<n;t++)
+        {
+            if(arr[t]!=j+k*t)
+                ti+=1;
+        }
+        p[j].first=ti;
+        p[j].second=j;
```

Comment:
Just a general *Nit* : Instead of making an extra array, sorting and taking the least value, why not just compare at each iteration to keep track of the minimum one :)